product_category_pricelist_button/__manifest__.py

```diff
@@ -0,0 +1,19 @@
+# © 2024 Solvos Consultoría Informática (<http://www.solvos.es>)
+# License LGPL-3.0 (https://www.gnu.org/licenses/lgpl-3.0.html)
+
+{
+    "name": "Product category pricelist button",
+    "summary": """
+        Adds a smart button to product categories linking to associated pricelists.
```

Comment:
Pending items reference:
```suggestion
        Adds a smart button to product categories linking to associated pricelist items.
```